addons/sale/models/sale_order.py

```diff
@@ -246,6 +246,9 @@ def _rec_names_search(self):
         compute='_compute_invoice_status',
         store=True)
 
+    sale_warning_text = fields.Text('Sale Warning', help="Internal warning for the partner or the products as set by the user.",
+                                    compute='_compute_sale_warning_text')
```

Comment:
```suggestion
    sale_warning_text = fields.Text(
        string="Sale Warning",
        help="Internal warning for the partner or the products as set by the user.",
        compute='_compute_sale_warning_text',
    )
```
Keep the line length < 100 characters and also use double quoted string for the `string` attribute as it's a label displayed to the user.